torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1017,7 +1023,7 @@ bool InsertObserversHelper::valueNeedsToBeQuantized(Value* v) {
   }
   // Check whether user is quantizable
   for (const auto& use : v->uses()) {
-    if (nodeQuantizable(use.user)) {
+    if (nodeQuantizable(use.user) && !isDynamicLSTMInputs(v, use, is_dynamic)) {
```

Comment:
this should be reversed, you need to check this is LSTM weight and quantize it
`nodeQuantizable(use.user) || isDynamicLSTMWeight(v, use, is_dynamic)`

Reply:
oh sorry, this should work as well

Reply:
`nodeQuantizable(use.user) || isDynamicLSTMWeight(v, use, is_dynamic)` would end up quantizing all input tensors of lstm.

Reply:
but still better to just check for weight I think, since there are other values when offset >= 2

Reply:
you can remove "lstm" from the the list in nodeQuantizable